Core/Functions/Core.lua

```diff
@@ -122,6 +122,7 @@ end
 function F.PixelPerfect()
   local perfectScale = 768 / E.physicalHeight
   if E.physicalHeight == 2160 then perfectScale = perfectScale * 2 end
+  if E.physicalHeight == 2880 then perfectScale = perfectScale * 2 end
```

Comment:
How about improving this a little bit?

```suggestion
  if E.physicalHeight == 2160 or E.physicalHeight == 2880 then
      perfectScale = perfectScale * 2
  end
```

In the future, if there are more resolutions than just 2160/2880, we could store them in a table and check the `E.physicalHeight` against that table.